internal/charger/easee.go

```diff
@@ -24,6 +24,17 @@ type Easee struct {
 	cache         time.Duration
 }
 
+// charge mode definition
+const (
```

Comment:
Du kannst du modi allerdings nicht hier deklarieren- Du bist im charger Namensraum! Daher ins Easee Package verschieben. Sonst Nameclash.

Reply:
ich kann sie auch einfach lowercasen, oder?

Reply:
Ignorier den Kommentar. Still learning how go package and visibility works.
Habs jetzt verschoben.